index.html

```diff
@@ -204,8 +243,8 @@ <h4>Skills</h5> <img class="arrowSep" src="./img/tech logos/arrow right.svg" alt
             </section>
 
 
-            <section class="contactMe">
-
+            <section id="contactMe" class="contactMe">
+<div id="contentContainer">
                 <h2 id="contactTitle">Contact me</h2>
                 <p class="aboutP" id="contactP">If you have an application you are interested in developing, a feature
                     that you need built or a project that needs coding.
```

Comment:
- [ ] Please make sure that you remove the vertical overflow of these elements in your contact section to make your design match the Figma design. :pray: 

  
  ### Your design 
  ![image](https://github.com/DamianDib9/personal-portfolio/assets/73854999/f805a517-1815-4979-81fc-81d570c51225)
  
  ### Figma design
  ![image](https://github.com/DamianDib9/personal-portfolio/assets/73854999/969d115f-512f-4f16-ba88-a16b11c8fe41)

Reply:
![image](https://github.com/DamianDib9/personal-portfolio/assets/121461459/8ef03635-d812-40ec-8fae-9909f22a1b77)

OK! thanks for your kind review.
Fixed!